cmd/ocm-backplane/login/login.go

```diff
@@ -494,3 +512,11 @@ func getBackplaneCNAME(backplaneURL string) (string, error) {
 	}
 	return resolution, nil
 }
+
+// isValidNamespace validates the input string against  Kubernetes namespace rules.( RFC 1123 )
+func isValidKubernetesNamespace(namespace string) bool {
+	// RFC 1123 compliant regex pattern)
+	pattern := `^[a-z0-9]([-a-z0-9]*[a-z0-9])?$`
+	matched, _ := regexp.MatchString(pattern, namespace)
+	return matched
```

Comment:
```suggestion
	return regexp.MustCompile(pattern).MatchString(namespace)
```
nit: fewer variable assignments >>